modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -220,6 +221,54 @@
             .nextUntil('.section-field').hide()
     }
 
+    /*
+     * Bind validation checks on available fields, triggered when
+     * the fields is blurred or changed
+     */
+    FormWidget.prototype.bindValidation = function () {
+        let fields = this.getFieldElements(),
+            form = this.$form;
+
+        for (let field of fields) {
+            let fieldClasses = field.className.split(/\s+/).map(function (className) {
+                var fieldClass = className.match(/^([a-z\-]+)-field$/);
+                if (fieldClass) {
+                    return fieldClass[1];
+                } else {
+                    return null;
+                }
+            }).filter(function (className) {
+                return (className != null);
+            });
+
+            if (fieldClasses.length === 0) {
+                continue;
+            }
+
+            // Load validator depending on field type
+            switch (fieldClasses[0]) {
+                case 'text':
+                case 'password':
+                    this._validateText(form, field);
+                    break;
+            }
+        }
+    };
+
+    FormWidget.prototype._validateText = function (form, field) {
+        let ele = this.$el;
+
+        field.querySelector('input').addEventListener('blur', function (ev) {
+            $(ele).request('onValidateField', {
```

Comment:
No need to pass `ele` to `$()`, it's already a jQuery object, could just do `$elem.request` directly